devtools/fixture_collector.py

```diff
@@ -79,14 +110,149 @@ async def main() -> None:
                 )
             )
 
-    print(f"Fixtures written to {target_dir}")
+    if not clean or verbose:
+        print(f"Fixtures written to {target_dir}")
 
     zip_file_name = f"{target_dir}.zip"
     with zipfile.ZipFile(zip_file_name, "w") as zip_file:
         for file_name in os.listdir(target_dir):
             zip_file.write(os.path.join(target_dir, file_name), file_name)
+            if clean:
+                os.remove(os.path.join(target_dir, file_name))
 
     print(f"Zip file written to {zip_file_name}")
 
+    if clean:
+        try:
+            os.rmdir(target_dir)
+            if verbose:
+                print(f"Removed {target_dir}")
+        except OSError as err:
+            print(f"Could not clean folder: {err.strerror}")
+        except FileNotFoundError:
+            pass
+
+
+def _read_ha_config(file_path: str) -> dict[str, list[str | None]]:
+    result: dict[str, list[str | None]] = {}
+    try:
+        with open(file_path) as fp:
+            content = json.load(fp)
+    except (FileNotFoundError, ValueError):
+        return result
+
+    if content:
+        for entry in content["data"]["entries"]:
+            if entry["domain"] != "enphase_envoy" or entry["source"] == "ignore":
+                continue
+            data = entry["data"]
+            unique_id = entry["unique_id"]
+            result[unique_id] = [
+                data["host"],
+                data["username"],
+                data["password"],
+                data["token"],
+            ]
 
```

Comment:
Made a small tweak here to reduce the code

Reply:
Please double check I didn't break anything

Reply:
done, works fine, thanks